src/pages/ActionsPage.js

```diff
@@ -23,44 +24,47 @@ function ActionsPage() {
 
   return (
     <Swipeable onSwipedRight={handleRightSwipe}>
-      <MyGrid
-        container
-        direction="column"
-        alignContent="stretch"
-        alignItems="center"
-        justify="space-between"
-      >
-        <Grid item>
-          <Typography variant="h1" align="center">
-            Looks like you're finished.
-          </Typography>
-        </Grid>
-        <Grid item>
-          <Typography variant="h2" align="center">
-            What would you like to do now?
-          </Typography>
-        </Grid>
-        <Grid item>
-          <Button component={Link}>Print</Button>
-        </Grid>
-        <Grid item>
-          <Button component={Link}>Download</Button>
-        </Grid>
-        <Grid item>
-          <Button data-cy="ReadMode" component={Link} to="/view-mode">
-            Read Mode
-          </Button>
-        </Grid>
-        <Grid item>
-          <Button component={Link} to="/">
-            Start Over
-          </Button>
-        </Grid>
-        <Grid item>
-          {/* <BottomNavBar {...{ pageNumber, totalPages }} /> */}
-          <BottomNavBar totalPages={totalPages} />
-        </Grid>
-      </MyGrid>
+      <Div100vh>
+        <MyGrid
+          container
+          direction="column"
+          alignContent="stretch"
+          alignItems="center"
+          justify="space-between"
+        >
+          <Grid item>&nbsp;</Grid>
```

Comment:
is there a reason why &nbsp; is there?

Reply:
uuuu i didn't know you could comment here